02/app.js

```diff
@@ -1,7 +1,29 @@
 
 /* rozwiązanie z pętlą for */
-const x = 4;
+const x = prompt('Podaj liczbę!')
 
+let i = 0;
+for (i = 1; i <= 9; i++) {
+    let result = x * i;
+    console.log(`${x} x ${i} = ${result}`)
+}
 
+/* rozwiązanie z pętlą while  */
 
-/* rozwiązanie z pętlą while  */
\ No newline at end of file
+const a = prompt('Podaj liczbę!');
+const n = 4;
```

Comment:
Zadanie powinno działać jeśli zmienię `n = 4` na `n = 5` - niestety obecnie nie zadziała ;)

PS. Wynik jest ok, ale to co jest przed znakiem `=` się nie zgadza :)